integration-tests/kotlin-dsl/pom.xml

```diff
@@ -100,6 +124,44 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>native</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.package.type>native</quarkus.package.type>
+            </properties>
+            <build>
+                <plugins>
+                    <!-- remove me -->
```

Comment:
I guess this can be removed to reinstate the sanity-checks execution.

Reply:
Oops nice catch 👍